pkg/client/openshift/buildconfig.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 const (
-	checkBcRetries         = 4
+	checkBcRetries         = 30
```

Comment:
This is the issue that @radtriste found where the CLI was not waiting enough time to check whether the BuildConfig was present. Using 30 retries (which means 2 minutes in total) works fine.